mail/views.py

```diff
@@ -237,3 +238,37 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class MailWebhookView(APIView):
+    """
+    View class that handles HTTP requests to capture bounced email
+    """
+    permission_classes = (MailGunWebHookPermission, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
+        """
+        POST method handler
+        """
+        event = request.POST.get("event", None)
+        recipient = request.POST.get("recipient", None)
+        error = request.POST.get("error", None)
+        message_headers = request.POST.get("message-headers", None)
+        log_error_on_bounce = request.POST.get("log_error_on_bounce", False)
+        error_msg = (
+            "Webhook event {event} received by Mailgun for recipient {to}: {error}".format(
+                to=recipient,
+                error=error,
+                event=event
+            )
+        )
+
+        if isinstance(log_error_on_bounce, str):
+            log_error_on_bounce = True if log_error_on_bounce == "True" else False
+
+        if log_error_on_bounce is True and event == "bounced":
```

Comment:
Instead of checking if it's a string above, please change this to `if log_error_on_bounce.lower() == "true" and event == "bounced"`